luigi/metrics.py

```diff
@@ -66,6 +66,9 @@ def handle_task_disabled(self, task, config):
     def handle_task_done(self, task):
         pass
 
+    def handle_task_statistics(self, task, statistics):
+        pass
```

Comment:
Is there a common implementation that can be introduced here? Or within the Datadog and/or Prometheus implementations of `MetricsCollector`?

If not, and this method needs to remain as an optionally-implemented method, i would ask at a minimum for there to be appropriately informative and discoverable documentation written surrounding its intended use. (I don't believe there are currently any `docs/` content around metrics collectors).

Reply:
This method is entirely optional. I had not provided any documentation and tests yet, because I was hoping to gather some feedback in https://github.com/spotify/luigi/issues/3342 as you suggested in my earlier pull request (https://github.com/spotify/luigi/pull/3333#issuecomment-2591417339).

I will gladly add tests and documentation if you are convinced this change is small enough.